sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -871,7 +1100,7 @@ private boolean isManagementToken(String lockToken) {
 
     private Mono<Void> updateDisposition(MessageLockToken message, DispositionStatus dispositionStatus,
         String deadLetterReason, String deadLetterErrorDescription, Map<String, Object> propertiesToModify,
-        String sessionId) {
+        String sessionId, AmqpTransaction transactionId) {
```

Comment:
Instead of creating a new object for every time (AmqpTransaction), why not pass the ServiceBusTransactionContext in here and pass the parameter onto the ServiceManagementNode?.